content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/moves_history.rst

```diff
@@ -0,0 +1,99 @@
+=============
+Moves history
```

Comment:
Just a general thought: Should we name these docs like "Moves history" or Moves history report" ? What do you think

Reply:
My thought was that since this doc is already located inside the reporting subsection, we could keep "report" out of the title. But, I'm open to other ideas, if other reviewers (@odoo/us-doc-review) would like to share their thoughts and input? (:

Reply:
It's a good question, and ultimately up to you I think, Felicia.

FWIW, my vote is we *do* add the word "report" to the title (or other similar qualifying nouns, like "analysis") but *only* when we are referring to or walking through a _very specific_ type of report. 

I think this way, because: 
- despite it being located under a **Reporting** subsection, it still might not be obvious to users what they're looking at especially in the cases where they land on the doc having not used the toctree to get there (e.g. a support ticket link, search engine, etc). 
- There is existing precedent to do this, like we see in a couple CRM sections (below)
![Screenshot 2024-05-16 at 6 10 27 PM](https://github.com/odoo/documentation/assets/5496809/4fcd4ec3-d51b-49a8-8821-78e70f69af35)
- When it doubt do the redundant/exhaustive thing, as to eliminate any possible confusion 😄 

However, for this doc, (unless I'm mistaken) I don't think we're actually covering specific _report_, per se, as much we are covering the UI of a specific _dashboard_, to which we can obtain multiple types of reports _from_. This is the same setup as CRM, where through the _Pipeline_ dashboard, many many reports can be generated 🤯 

For the _Moves History_ dashboard, we could, for example, generate an incoming stock report (screenshot below), which would be helpful for warehouse workers in knowing what locations need to be organized/prepared for the incoming stock. I also think we could do a traffic report with this dashboard, as another example, to help cut down on unauthorized moves, theft, etc. There are likely much more reports we could pull from this one dashboard!

![image](https://github.com/odoo/documentation/assets/5496809/f122ec67-fdb6-4820-9f31-48e3fa3f1c69)

Reply:
```suggestion
Moves history dashboard
```